source/reference/mongosync-states.txt

```diff
@@ -85,3 +85,15 @@ operations in that state.
      - - ``GET`` :ref:`/progress <c2c-api-progress>`
        - ``GET`` :ref:`/reverse <c2c-api-reverse>`
 
+   * - .. _c2c-state-reversing:
+
+       ``REVERSING``
+
+     - The sync process copies metadata from the destination
+       cluster to the source cluster. When complete, the phase
+       reverts to applying change events and the original source
+       cluster becomes the new destination cluster while the old
+       destination becomes the new source.
```

Comment:
Not sure about the use of "revert" here. Would it be accurate to say something like:

```suggestion
     - The sync process copies metadata from the destination
       cluster to the source cluster. Then, MongoDB swaps the
       source and destination clusters and resumes applying
       change events.
```

I don't love the use of "swap" in my example from a technical standpoint, but if possible I'd prefer a shorter way of saying that source and destination switch places.